tools/create_collection/create_collection.xml

```diff
@@ -60,26 +60,26 @@
 
 .. class:: warningmark
       
-*NOTE*: Be advised, to use the in workflows. And prioritize the use of the normal 'checkbox in dataset history' tool otherwise.
+*NOTE*: Be advised, to use this tool in workflows. And prioritize the use of the normal 'checkbox in dataset history' tool otherwise.
```

Comment:
This note is redundant with the warning in "what it does" (plus not really meaningful)